components/recipesList/recipesList.tsx

```diff
@@ -1,9 +1,25 @@
 import React from 'react'
 
 const RecipesList = () => {
+
+  const RecipeArrayList = [
+
+    { name: 'Chicken Tikka Masala', effort: '2 Stunden', },
+    { name: 'Indisches Butter Chicken aus dem Ofen', effort: '2 Stunden', },
+    { name: 'Börek mit Käse und Spinat', effort: '2 Stunden', }
+    { name: 'Georgische Hatschapuri', effort: '3 Stunden', }
+    { name: 'Krautwickel mit Reis und Hackfleisch', effort: '4 Stunden', }
+
+  ]
+
   return (
     <div>
-      Hier ist die Liste der Rezepte
+      Hier ist die Liste der Rezepte:
+      <ol>
+        {RecipeArrayList.map((recipe) =>
```

Comment:
```suggestion
        {RecipeArrayList.map((recipe, index) =>
       
```

add index to map list

Reply:
done